tests/sentry/workflow_engine/endpoints/test_organization_detector_index.py

```diff
@@ -496,6 +496,56 @@ def test_query_by_assignee_invalid_user(self) -> None:
         )
         assert len(response.data) == 0
 
+    def test_query_by_project_owner_user(self) -> None:
+        new_project = self.create_project(organization=self.organization)
+        detector = self.create_detector(
+            project_id=new_project.id, name="Test Detector", type=MetricIssue.slug
+        )
+
+        owner = self.create_user()
+        self.create_member(
+            user=owner,
+            role="owner",
+            organization=self.organization,
+        )
+        self.login_as(user=owner)
+
+        # Verify that the owner can see detectors across all projects
```

Comment:
isn't this just checking one project though by passing the new project id?

Reply:
I can change the wording of the comment, but the intent of the test is to verify that the owner can access detectors for projects that they are not members of (in this test the owner is not a member of the project being filtered on)